runner/query_run.go

```diff
@@ -80,7 +80,12 @@ func runQueryOnDatabase(ctx context.Context, server *state.Server, collectionOpt
 	}
 	defer db.Close()
 
-	if postgres.StatsHelperExists(ctx, db, "explain_analyze") {
+	h, err := postgres.NewCollectionHelper(ctx, logger, server, collectionOpts, db)
```

Comment:
Maybe consider adding a different constructor or argument here to initialize without the data that's not needed (like Postgres version in this case).

Reply:
After thinking about it some more I think it'd be best if we always fetch version and role information (i.e. not make it conditional).

We'll need roles soon enough (to check helper function ownership), and whilst we don't need version in this code path right now, I could anticipate that changing (e.g. to conditionally set certain EXPLAIN flags).